### ISSUE TITLE
Warning should not be raised for beh.tsv not accompanied by a events.tsv

### ISSUE THREAD

**Issue by Remi-Gau** [bids-standard/legacy-validator#2058]
Modified the synthetic bids example to only have:

```
synthetic
├── dataset_description.json
├── README
└── sub-01
    └── ses-01
        └── beh
            └── sub-01_ses-01_task-stroop_beh.tsv
```

deno validator (v1.14.7-dev.0-249-g93ca7a02) gives

```
bids-validator  synthetic 

...

        [WARNING] Task scans should have a corresponding 'events.tsv' file.
If this is a resting state scan you can ignore this warning or rename the task to include the word "rest".
 (EVENTS_TSV_MISSING)

                ./sub-01/ses-01/beh/sub-01_ses-01_task-stroop_beh.tsv
                        Evidence: schema.rules.rules.checks.events.EventsMissing

                1 more files with the same issue

        Please visit https://neurostars.org/search?q=EVENTS_TSV_MISSING for existing conversations about this issue.

...
```
